src/Restore.php

```diff
@@ -176,17 +176,20 @@ public function restoreConfigs(array $skipComponents = []): void
 
                 if (in_array($metadataFilePath, $componentConfigurationsFiles)) {
                     $metadataData = json_decode((string) $this->getDataFromStorage($metadataFilePath), true);
-                    array_walk($metadataData, function (&$v): void {
-                        unset($v['id']);
-                        unset($v['timestamp']);
-                    });
 
-                    $branchAwareComponents = new Components($this->branchAwareClient);
+                    if (!empty($metadataData)) {
```

Comment:
přidal jsem jen if na !empty
job spadl na chybě `Argument must be a non-empty array of metadata objects`

Reply:
hm, a vies preco sa to deje?
Ak subor existuje, tak by mal byt validny, nie?
Das mi prosim viac info?

Reply:
ten soubor se zazálohuje s prázdným polem
asi bych to mohl odfiltrovat i v backup appce aby se ten soubor vůbec nevytvořil... ale nechal bych tu i tu

Reply:
@webrouse stejnou kontrolu jsem přidal do backup appky - https://github.com/keboola/php-kbc-project-backup/pull/9

Reply:
ok, uz rozumiem.